02-express-tutorial/controllers/people.js

```diff
@@ -0,0 +1,68 @@
+const { people } = require("../data.js");
+
+const addPerson = (req,res) => {
+    const { name } = req.body
+    if (name) {
+        people.push({ id: people.length + 1, name: req.body.name });
+        res.status(201).json({ success: true, name: req.body.name });
+        return console.log(name)
+    }
+    else{
+        res.status(400).json({ success: false, message: "Please provide a name" })
+        return console.log(`No name was entered`)
+    }
```

Comment:
This controller looks really great!

Also, just wanted to quickly point out that if you do have a `return` in an `if` statement, you don't need the `else` statement (necessarily) because of the `return` (there's also nothing wrong with it). Just something to consider when finding the style of JS you like to write